src/components/Badge/Badge.styles.ts

```diff
@@ -101,96 +101,65 @@ export const StyledBadge = styled(Box, {
         props: ({ variant }) => variant === BadgeVariant.Default,
         style: {
           pointerEvents: 'none',
-          backgroundColor: (theme.vars || theme).palette.surface1.main,
-          color: (theme.vars || theme).palette.alphaLight900.main,
-          ...theme.applyStyles('light', {
-            color: (theme.vars || theme).palette.alphaDark900.main,
-          }),
+          backgroundColor: (theme.vars || theme).palette.badgeDefaultBg,
```

Comment:
Since we're reusing this pattern a lot, what if we defined:

```ts
const palette = (theme.vars || theme)
```

at the top of the function def?

Reply:
https://lifi.atlassian.net/browse/LF-14824